gopa.yml

```diff
@@ -83,6 +83,9 @@ modules:
         parameters:
           compress_enabled: true
           bucket: "Snapshot"
+          snapshottime_toless: "1440,720,360,180,90,45,20,10,1"
```

Comment:
Hi, @ycbydd 
These config are not human friendly, can you switch to use time.duration format, here is the reference:
https://golang.org/src/time/format.go?s=36330:36376#L1222